.github/workflows/pr-merge.yml

```diff
@@ -0,0 +1,69 @@
+name: TNP PR Merged
```

Comment:
In the someway we called the other workflow "publish preview", let's call this 'publish release"

Reply:
Updated